docs/README.md

```diff
@@ -1,15 +1,30 @@
----
-editLink: true
----
-
-# Welcome to the open DC grid working group
-
-This document specifies core requirements and suggestions for implementation of a DC grid to interconnect multiple energy sources and sinks. It can be considered a grid code for a low-power separated extra-low voltage (SELV) DC grid.
-
-The specification is manufacturer-agnostic. Only the interfaces at grid level will be defined to allow seamless integration of devices made by different manufacturers.
+# Open DC Grid
 
 <center>
 
 ![Grid Overview](./images/grid-overview.svg)
 
 </center>
+
+## DC Microgrid
+The Open DC Grid is a microgrid architecture to permit devices to exchange electric power using DC (direct current). The architecture is defined by the [Open DC Grid Standard](./standard), a document than can be freely accessed and used by anyone at no charge.
+The architecture defined by the standard permits devices to be connected using several kinds of link technology but the most commonly used link is an electrical bus operating at a nominal 48 V. 
+The 48V bus link is wired using a configuration referred to as separated extra-low voltage (SELV) and is designed to be inherently safe for humans to touch in most environments.
+The Open DC Grid is particularly well suited for distributing power in Solar Home Systems often used in the developing world when normal AC power is unavailable or unaffordable.
+
+## Smart Grid
+Devices in the Open DC Grid communicate to optimize the use of the limited power typically available in a remote microgrid. Appliances identify themselves and the available power is assigned on a priority basis by the owner or grid manager.
+Power sources coordinate to source power reliably when connected to a common bus. Individual homes with a Solar Home System can choose to share power with their neighbors under terms defined by the grid manager.
+
+The grid provides basic communications with a protocol hierarchy defined by the Standard. The upper layers are device independent. The physical layers depend on the type of link. 
+An Open DC Grid micgrogrid can run completely stand-alone without connection to the outside world. Bridges to standard Internet protocols can potentially be used to link devices to the Internet for remote monitoring and management.
+The Standard defines low level protocols to support billing mechanisms such as pay-as-you-go popular in the developing world but depends on reseller provided software to incorporate these low-level protocols
```

Comment:
Not sure if we should use the term "developing world". I try to avoid it, as it is a) very vague and b) suggests that our western economic model is what is best for everyone. Maybe we can use energy access sector instead in this case?